docs/core/reference/changelog.md

```diff
@@ -10,6 +10,26 @@ For updates to subpackages, see below:
 - [Node.js SDK](https://github.com/chain/chain/blob/1.1-stable/sdk/node/CHANGELOG.md)
 - [Ruby SDK](https://github.com/chain/chain/blob/1.1-stable/sdk/ruby/CHANGELOG.md)
 
+<a name="1.1.5"></a>
```

Comment:
What's your take on how to deal with backported doc changes? Should we bother with them at all?